packages/block-library/src/post-comments-count/edit.js

```diff
@@ -17,6 +17,18 @@ import apiFetch from '@wordpress/api-fetch';
 import { addQueryArgs } from '@wordpress/url';
 import { __ } from '@wordpress/i18n';
 
+const NoCommentsWarning = ( { style, ...props } ) => {
+	// Text decoration is stripped as it can't be reset as per other styles.
+	const { textDecoration, ...blockStyles } = style || {};
```

Comment:
This appears to be working nicely for me. We can arguably remove the styles that we don't want to pass down, or only select those styles that we do want to pass to the warning wrapper. Since this particular component only appears to be adversely affected by the textDecoration value, the approach here looks good to me 👍

Reply:
I wasn't 100% sure which way to go here but leant towards the cleaner option here given the CSS resets are already in place and doing their job.